contracts/mocks/compound/CompTimelock.sol

```diff
@@ -63,7 +63,7 @@ contract CompTimelock {
     address public pendingAdmin;
     uint256 public delay;
 
-    mapping(bytes32 => bool) public queuedTransactions;
+    mapping(bytes32 txHash => bool queued) public queuedTransactions;
```

Comment:
That is a mock, so I don't really care, bu I think we should not use txHash for anything that is not an actual txHash

Reply:
I renamed this to `operationHash` instead because I think it reflects better, but let me know if we need to rollback it or you have another proposal.